frontend/app/src/store/assets/retrieval.ts

```diff
@@ -43,7 +43,7 @@ export const useAssetInfoRetrieval = defineStore(
         const address = getAddressFromEvmIdentifier(id);
         return `EVM Token: ${address}`;
       }
-      return '';
+      return id;
```

Comment:
when does this happen? (ideally we should never show raw identifiers to the user)

Reply:
this happen as fallback if we pass unknown identifier to <amount-display>, for example we set asset="Gwei", it will show "Gwei" in the <amount-display>

Reply:
Maybe keep it empty string and thing of another approach for this the `Gwei` scenario?